src/utils/validateMediaAndText/validadeText/validadeText.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { validateTextLength } from './validateText';
+
+describe('validate texts', () => {
+  it('validate text length', () => {
```

Comment:
Seu `describe` e `it` devem conversar, ou seja, quando o teste rodar precisamos ler algo como `Text validation limit text length` sendo `describe('Text validation'` e `it('limit text length'`